core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py

```diff
@@ -59,14 +54,25 @@ def is_valid_config(self) -> bool:
             return False
 
     def command_list(self) -> List[Union[str, pathlib.Path]]:
+        """List of arguments to be used in the command line call.
+        Refer to https://thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html for details about each argument."""
```

Comment:
nice comment

Reply:
It's difficult to find well written documentations like this one.